Utils/Dataflow/README

```diff
@@ -133,8 +133,34 @@ following requirements:
 
 [3] EOP, End-of-Process marker. Default value: '\0' (NULL)
 
+==================
+* 3. Stage types *
+==================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.).
+
+2. Transform stage - these stages perform a set of operations for different
+cases on metadata and form output messages according to a particular pattern
+(e.g. select relevant metadata from a raw dataset, append metadata
```

Comment:
Sorry I missed related conversation above(https://github.com/PanDAWMS/dkb/pull/278#discussion_r304831028), so I resolve this conversation and will add a coment there.

Reply:
@mgolosova And you asked (and I even answered ;) ), but there was a difference between definitions.
From my point of view, it was something like "not already transformed, but uploaded with any source stage data", I mean "data from an external source as it was", but I see -- in terms of ATLAS-related dataflow, it is a completely different thing. I suppose I can reword it to prevent misunderstanding.